x/gov/module.go

```diff
@@ -154,6 +148,14 @@ func NewAppModule(
 	}
 }
 
+var _ appmodule.AppModule = AppModule{}
```

Comment:
I think for clarity purposes, I would add all the `var _ module.Has* = AppModule{}` in this file. But we can do it in a follow-up

Reply:
Let's do in follow-ups